requirements.txt

```
@@ -1,4 +1,4 @@
 tifffile>=2021.7.2,<2022.4.28
 numpy
 pyproj
-zarr>=2.10.0
+zarr==2.12.*
```

On second glance, this could break some projects downstream. Can I suggest an alternative?

```suggestion
zarr>=2.12.0,<2.13
```